doc/changelog.rst

```diff
@@ -8,14 +8,17 @@ To be released at some future point in time
 
 Description
 
+- Added developer documentation on how to run a single test case
```

Comment:
Do you want to add a blurb about removing the two `SMARTREDIS_*` vars just in case someone was using them?

Reply:
That's a fair point, will do